packages/hops-mixin/README.md

```diff
@@ -0,0 +1,70 @@
+# `hops-mixin`
+
+[![npm](https://img.shields.io/npm/v/hops-mixin.svg)](https://www.npmjs.com/package/hops-mixin)
+
+Contains the building blocks needed to build your own hops mixins. Mixins allow you to alter and extend Hops' functionality.
+
+Tap into them using your own mixins and [reconfigure Webpack](https://github.com/xing/hops/blob/master/packages/styled-components/mixin.core.js#L4), [register additional Express](https://github.com/xing/hops/blob/master/packages/development-proxy/mixin.core.js#L6) middlewares and [Yargs commands](https://github.com/xing/hops/blob/master/packages/graphql/mixin.core.js#L5) or [fetch data to bootstrap your React application](https://github.com/xing/hops/blob/master/packages/react/server-data/mixin.server.js#L11).
+
+### Installation
+
+```bash
+$ yarn add hops-mixin
+# OR npm install --save hops-mixin
+```
+
+If you don't already have an existing Hops project read this section [on how to set up your first Hops project.](https://missing-link-explain-quick-start)
+
+### API
+
+#### Mixin
+
+Base class to extend from when building a mixin. Ensures the config is available in `this.config`.
+
+```javascript
+const { Mixin } = require('hops-mixin');
+
+class MyMixin extends Mixin {}
+```
+
+#### strategies
+
+Strategies allow to define mixin hooks that are usable by other mixins. There are various types of strategies. `callable` will make the method available to other mixins. Methods with strategy `pipe` pass each implementation's output to the next, using the first argument as the initial value. All other arguments are being passed to all implementations as-is.
+
+For a complete list of available strategies, have a look at the [mixinable documentation](https://github.com/untool/mixinable).
+
+##### Callable example
+
+A mixin that exposes a method to retrieve the build config.
+
+```javascript
+const {
+  Mixin,
+  strategies: { sync: callable },
+} = require('hops-mixin');
+
+class BuildConfigMixin extends Mixin {
+  getBuildConfig() {
+    return this.buildConfig;
+  }
+}
+
+MyMixin.strategies = {
+  getBuildConfig: callable,
+};
+```
+
+Other mixins are now able to call the method.
+
+```javascript
+const {
+  Mixin,
+  strategies: { sync: callable },
```

Comment:
We should remove this line here

Reply:
I think we still need it here, I removed it from the other ConsumerMixin example, though, because it was unnecessary there.